### PR TITLE
API-9701 Add response_time and first_response_time reports

### PR DESCRIPTION
**Pull request by janowsiany**
- [Jira](https://livechatinc.atlassian.net/browse/API-9701)

### CODE REVIEW

**Review by MaciejSzaflik — APPROVED** [review]
looks good to me 👍

File: src/pages/data-reporting/reports-api/v3.4/index.mdx [hunk 1]
```diff
@@ -80,6 +80,10 @@ You can authorize your calls to the Reports API using one of the following metho
...
+| `filters.agent_response.exists`                                       | No       | `bool`    |                                                                                                                                                                                                                                             |
+| `filters.agent_response.first`                                        | No       | `bool`    | Modifier that makes the other `agent_response` filters operate on first response time.                                                                                                                                                      |
+| `filters.agent_response.agents.<string_filter_type>`<sup>**2**</sup>  | No       | `any`     | Described below.                                                                                                                                                                                                                            |
```
Comment (oliwiapolec) [inline src/pages/data-reporting/reports-api/v3.4/index.mdx:85]:
```suggestion
| `filters.agent_response.agents.<string_filter_type>`<sup>**2**</sup>  | No       | `any`     | Described below. Works only if the `agent_response.first` filter is set to `true`.                                                                                                                                                                                                                           |
```

File: src/pages/data-reporting/reports-api/v3.4/index.mdx [hunk 2]
```diff
@@ -80,6 +80,10 @@ You can authorize your calls to the Reports API using one of the following metho
...
+| `filters.agent_response.groups.<integer_filter_type>`<sup>**3**</sup> | No       | `any`     | Described below.                                                                                                                                                                                                                            |
```
Comment (oliwiapolec) [inline src/pages/data-reporting/reports-api/v3.4/index.mdx:86]:
```suggestion
| `filters.agent_response.groups.<integer_filter_type>`<sup>**3**</sup> | No       | `any`     | Described below. Works only if the `agent_response.first` filter is set to `true`.                                                                                                                                                                                                                            |
```

File: src/pages/messaging/agent-chat-api/v3.4/index.mdx [hunk 3]
```diff
@@ -529,6 +530,16 @@ You can pass only one of the following values at a time: `values` or `exclude_va
...
+**\*\*\*\*\*\*)**
+`<agent_response_filter_type>` can take the following values:
+
+- `exists` (`bool`)
+- `first` (`bool`) - modifier that makes the other `agent_response` filters operate on first response time
+- `agents.values` (`string[]`) - an array of agent IDs
+- `agents.exclude_values` (`string[]`) - an array of agent IDs
+- `groups.values` (`int[]`) - an array of group IDs
+- `groups.exclude_values` (`int[]`) - an array of group IDs
+
```
[the remark this answers is not in the document]
Comment (janowsiany) [reply src/pages/messaging/agent-chat-api/v3.4/index.mdx:542]: Added

**Review by oliwiapolec — APPROVED** [review]

**Pull request merged**